mozdownload/scraper.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, destination=None, platform=None,
         logging.basicConfig(format=' %(levelname)s | %(message)s')
         self.logger = logging.getLogger(self.__module__)
         self.logger.setLevel(log_level)
+        logging.getLogger('redo').setLevel(logging.INFO)
```

Comment:
I would use the same log level as for mozdownload. So IMO using `log_level` here would be appropriate. I wonder if we should do the same for requests at the same time.